server/services/parcelService.js

```diff
@@ -2,6 +2,10 @@ const parcels = require('../../data/parcels.json')
 
 module.exports = {
   get: function () {
-    return parcels.map(parcel => ({ ref: parcel.parcelRef }))
+    return parcels
+  },
+  getByRef: function (parcelRef) {
+    const matches = parcels.filter(parcel => parcel.parcelRef === parcelRef)
```

Comment:
This now returns null, just need to replace `parcel.parcelRef` with `parcel.ref`

Reply:
I can fix in my branch if you like?